### PR TITLE
chore: fix lint

### PR DESCRIPTION
**Pull request by vados-cosmonic**
## Feature or Problem

Fix minor lint and test issues in `wash`

## Testing

Built on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

Tested on platform(s)
- [x] x86_64-linux
- [ ] aarch64-linux
- [ ] x86_64-darwin
- [ ] aarch64-darwin
- [ ] x86_64-windows

### Manual Verification

Verified manually locally

### CODE REVIEW

File: crates/wash-lib/src/start/wasmcloud.rs [hunk 1]
```diff
@@ -129,13 +129,13 @@ where
...
 /// # Examples
 ///
-/// ```no_run
+/// ```rust,ignore
 /// # #[tokio::main]
 /// # async fn main() {
 /// use wash_lib::start::download_wasmcloud_for_os_arch_pair;
 /// let os = std::env::consts::OS;
 /// let arch = std::env::consts::ARCH;
-/// let res = download_wasmcloud_for_os_arch_pair(os, arch, "v0.63.0", "/tmp/wasmcloud/").await;
```
Comment (brooksmtownsend) [inline crates/wash-lib/src/start/wasmcloud.rs:138]: That's weird, this function isn't even aptly named anymore 🤔

**Review by brooksmtownsend — DISMISSED** [review]

File: crates/wash-lib/src/start/wasmcloud.rs (hunk 1 above)
Comment (vados-cosmonic) [reply crates/wash-lib/src/start/wasmcloud.rs:138]: yeah super weird... honestly I was more surprised there were lints at all! we definitely CI for this...?

File: src/main.rs [hunk 2]
```diff
@@ -233,7 +233,7 @@ enum CliCommand {
 #[tokio::main]
 async fn main() {
     use clap::CommandFactory;
-    if env_logger::try_init().is_err() {}
+    let _ = env_logger::try_init();
```
Comment (connorsmith256) [inline src/main.rs]: Nit: should we just `init()`? Since this is the first thing we do it's probably right to panic if a logger already exists 😜

**Review by connorsmith256 — DISMISSED** [review]

File: src/main.rs (hunk 2 above)
Comment (vados-cosmonic) [reply src/main.rs]: Hey we definitely can -- I'll make this change real quick

**Review by connorsmith256 — APPROVED** [review]

**Pull request merged**